tests/test_components.py

```diff
@@ -111,3 +111,88 @@ def test_random(self):
 
         # no ValidationError is raised
         self.assertTrue(components.MineralConcentrationComponent().serialize(data))
+
+
+class SpeciesComponentTestCase(unittest.TestCase):
+    def test_immune(self):
+        data = {
+            'type': 'species',
+            'name': 'Human',
```

Comment:
What if someone inputs "human" with lower case letters?

Reply:
For creating a species?  Nothing special, they'll just have an all-lowercase species name for the rest of the game.